docker-image/docker/logback.xml

```diff
@@ -37,11 +37,11 @@
 
 <!--   <logger name="com.datastax.driver.core.QueryLogger" level="TRACE"/> -->
 
-<!--   <logger name="org.jboss.weld" level="DEBUG"/> -->
+  <logger name="org.jboss.weld" level="DEBUG"/>
```

Comment:
If we leave `DEBUG` level logging on by default, we should leave a note to that effect in the README.

Reply:
I don't need to commit this..

Reply:
That's cool too.